forms/review.form.php

```diff
@@ -44,46 +44,7 @@
 		// if userEmail is empty, add error message to array to be displayed later
 		$error['userEmail'] = 'Please Enter you Email';
 	}
```

Comment:
@billythakidd04 these are the changes I pulled into my branch. https://github.com/eiman-kased/starbulls/pull/123#discussion_r617160183

Reply:
interesting... from where?

Reply:
I see... I'm not sure why this change was made. Let me review this pr really quick.

Reply:
Since I created this pr i cant request changes. can you please note that this needs to be fixed and return the php in the form?

Reply:
I dont want that change mistakenly slipping through and having to fix it later.